client/FriendsList.js

```diff
@@ -16,12 +16,24 @@ FriendsList.prototype.update = function(message) {
 		case 'friend-updated':
 			this.handleUpdate(message)
 			break
+		case 'friend-hiding':
+			this.handleFriendHiding(message)
+			break
+		case 'friend-stale':
+			new Toast(message.name+' Is no longer active')
+			this.handleUpdate(message)
```

Comment:
We can just illicit an update here because the message will have the stale flag being set.